content/influxdb/v2.0/write-data/developer-tools/csv.md

```diff
@@ -104,6 +90,14 @@ macOS users, to persist the `ulimit` setting, follow the [recommended steps](htt
 
 {{% /note %}}
 
+## Telegraf 
+
+Use CSV data format in Telegraf as a way to write CSV data to InfluxDB. 
+
+For more information, see: 
+- [CSV input data format](/telegraf/v1.19/data_formats/input/csv/)
```

Comment:
I think you need to separate the paragraph and the list with a blank line for the list to render correctly:

```suggestion
For more information, see: 

- [CSV input data format](/telegraf/v1.19/data_formats/input/csv/)
```